frontend/test/lit/test_multi_qubit_gates.py

```diff
@@ -63,19 +63,20 @@ def circuit():
 
 print(circuit.mlir)
 
-
-# CHECK-LABEL: public @jit_circuit
+"""
```

Comment:
```suggestion
"""
# TODO: The only reason we are using the braket.local.qubit device here
# is because this test was developed before having support for custom devices.
# We should replace instead create a custom device that has support for ISWAP
# and PSWAP (which I think are unsupported in lightning.qubit and hence why they
# would be decomposed.
```